readthedocs/audit/models.py

```diff
@@ -178,13 +191,31 @@ class AuditLog(TimeStampedModel):
         blank=True,
         null=True,
     )
+    data = models.JSONField(
+        null=True,
+        blank=True,
+        help_text=_("Extra data about the log entry."),
+    )
 
     objects = AuditLogManager()
 
     class Meta:
 
         ordering = ['-created']
 
+    def user_html_display(self):
```

Comment:
This could also be a templatetag or similar.

Reply:
wanted to include the user in the same paragraph of the description, but I'll put it back with the other metadata instead.